999-storage-import-zipped-vhds/azuredeploy.json

```diff
@@ -0,0 +1,173 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.2105082",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "The location in which the resources should be deployed."
+            }
+        },
+        "Source": {
+            "type": "string",
+            "metadata": {
+                "description": "URL of a ZIP archive containing VHD images files. For example: https://provider.com/bin/support/azure-vhd.zip"
+            }
+        },
+        "Destination": {
+            "type": "string",
+            "metadata": {
+                "description": "URI of a Storage Account container with a writable SAS Token. For example: https://blob-storage-name.blob.core .windows.net/container-name?sp=racwl&st=2021-01-01T12:00:00&se=2021-02-01T12:00:00&spr=https&sv=2021-01-01&sig=123412341234"
```

Comment:
fix typo in uri

Reply:
The pipleline detected an issue with "core.windows.net" in the description, I added a space to split the url "core .windows.net" which look like a typo.

Reply:
Fixed

Reply:
IC - so we'll have to remove the hard-coded endpoint - you can link to a doc if you think it's necessary.

Reply:
Done.